test/potluck_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_is_an_empty_array
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_exists_couscous
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+
+    assert_instance_of Dish, couscous_salad
+  end
```

Comment:
this test shouldn't be here-- this is the potluck test, and you're testing dishes.  You've already tested in your dish test that a dish can exist, so this test is redundant and would be confusing to another developer looking at this code.